### PR TITLE
ACA-1640 load balancer with public ip

### PR DESCRIPTION
**Pull request by Yaish25491**
Adding alwayes statement to the first block in test_azure_load_balancer_with_public_ip test to verify the removal of load balancer and public IP resources

### CODE REVIEW

File: tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/main.yml [hunk 1]
```diff
@@ -12,6 +12,19 @@
...
 
+  always:
+    - name: Cleanup - make sure Load Balancer deleted, resource_group={{ resource_group }}
+      azure.azcollection.azure_rm_loadbalancer:
+        resource_group: "{{ resource_group }}"
+        name: "{{ load_balancer_name }}"
+        state: absent
+
+    - name: Cleanup - make sure Public IP deleted, resource_group={{ resource_group }}
+      azure.azcollection.azure_rm_publicipaddress:
```
Comment (nirarg) [inline tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/main.yml:23]: I think that deleting the resource group here is a mistake
This is the pre-created resource group, am I right?

**Review by nirarg — APPROVED** [review]
Looks good, thanks

**Comment by prabinovRedhat** [comment]
except of small suggestion, looks good

**Review by prabinovRedhat — APPROVED** [review]
lgtm

**Review by p3ck — APPROVED** [review]

**Review by prabinovRedhat — APPROVED** [review]

**Pull request merged**